packages/fast-usdc-contract/src/fast-usdc.flows.ts

```diff
@@ -132,3 +154,226 @@ export const forwardFunds = async (
   }
 };
 harden(forwardFunds);
+
+export interface ContextAdvance {
+  chainHub: ChainHub;
+  feeConfig: FeeConfig;
+  getNobleICA: () => OrchestrationAccount<{ chainId: 'noble-1' }>;
+  log: Console['log'];
+  statusManager: StatusManager;
+  usdc: { brand: Brand<'nat'>; denom: Denom };
+  zcfTools: { makeEmptyZCFSeat: () => ZCFSeat };
+  zoeTools: ZoeTools;
+}
+
+export const advanceFunds = (async (
+  orch: Orchestrator,
+  {
+    chainHub,
+    feeConfig,
+    getNobleICA,
+    log,
+    statusManager,
+    usdc,
+    zcfTools,
+    zoeTools: { localTransfer, withdrawToSeat },
+  }: ContextAdvance,
+  { evidence, risk }: EvidenceWithRisk,
+  config: {
+    notifier: SettlerKit['notifier'];
+    borrower: LiquidityPoolKit['borrower'];
+    poolAccount: OrchestrationAccount<{ chainId: 'agoric-any' }>;
+    settlementAddress: CosmosChainAddress;
+  } & CopyRecord,
+) => {
+  const feeTools = makeFeeTools(feeConfig);
+  const toAmount = (value: bigint) => AmountMath.make(usdc.brand, value);
+  const supportsCctp = makeSupportsCctp(chainHub);
+  // XXX async flow types
+  function asGuest<F extends HostAsyncFuncWrapper>(f: F): GuestOf<F> {
```

Comment:
since this is just a type fix, please accomplish through a type annotation. the function call looks like it's doing something.